### PR TITLE
[RFC: Docs] First draft of Navigator vs NavigatorIOS docs

### PR DESCRIPTION
**Pull request by brentvatne**

### CODE REVIEW

File: docs/NavigatorComparison.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- Under active development from the React Native team.
+- Written in JavaScript.
+- Works on iOS and Android.
+- Includes a simple navigation bar component similar to the default `NavigatorIOS` bar: `Navigator.NavigationBar`, and another with breadcrumbs called `Navigator.BreadcrumbNavigationBar`. See the UIExplorer demo to try them out and see how to use them.
+- You can provide your own navigation bar by passing it through the `navigationBar` prop.
+
+## NavigatorIOS
+
+- Small, limited API makes it much less customizable than `Navigator` in its current form.
+- Development belongs to OSS community - not used by the React Native team on their apps.
```
Comment (ide) [inline docs/NavigatorComparison.md]: OSS -> open-source, just to make it obvious

File: docs/NavigatorComparison.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  - A result of this is that there is currently a backlog of unresolved bugs, nobody who uses this has stepped up to take ownership for it yet.
+- Wraps UIKit, so it works exactly the same as it would on another native app. Lives in Objective-C and JavaScript.
```
Comment (ide) [inline docs/NavigatorComparison.md]: As a sub bullet point you might want to mention that you get animations and behavior that Apple has developed.

File: docs/NavigatorComparison.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+only work on the one platform. Below is an itemized list of differences
+between the two.
+
+## Navigator
+
+- Extensive API makes it completely customizable from JavaScript.
+- Under active development from the React Native team.
+- Written in JavaScript.
+- Works on iOS and Android.
+- Includes a simple navigation bar component similar to the default `NavigatorIOS` bar: `Navigator.NavigationBar`, and another with breadcrumbs called `Navigator.BreadcrumbNavigationBar`. See the UIExplorer demo to try them out and see how to use them.
```
Comment (ide) [inline docs/NavigatorComparison.md]: sub-bullet point: currently, animations are good but less refined than Apple's

**Comment by ide** [comment]
Covers most of it... thanks for writing this up.

**Comment by brentvatne** [comment]
Thanks for the review @ide! Made those changes, merging now.

**Pull request merged**